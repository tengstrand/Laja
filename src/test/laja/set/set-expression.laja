#write "target/test-output/set-expression.out"
  #set (name = "Mike")
  ## " must be escaped with \"
  #set (x = 
"Your name is:
  \"{name}\"")
{x}
  #set (age = "18")
Age:
  {age} years## This comment will remove the <newline> character(s).
#end