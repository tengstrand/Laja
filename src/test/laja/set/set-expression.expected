Your name is:
  "Mike"
Age:
  18 years